DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: tidyjson
 Title: Tidy Complex 'JSON'
-Version: 0.3.1.9000
+Version: 0.3.2.9000
```

Comment:
Do you mind bumping the version to 0.3.1.9001 instead? I will bump to 0.3.2 when I release to CRAN
```suggestion
Version: 0.3.1.9001
```